### PR TITLE
[Experimental] Updated SDK version to latest for adaptive dialog samples

### PR DESCRIPTION
**Pull request by chon219**
## Proposed Changes

  - Updated sdk version to `4.11.0` for adaptive dialog samples
  - Updated sample code to make sure everything work as expected
  - Renamed some file names (uppercase to lowercase) to make sure they can work on case sensitive file systems (e.g. linux).

### CODE REVIEW

File: experimental/adaptive-dialog/javascript_nodejs/01.multi-turn-prompt/package.json [hunk 1]
```diff
@@ -16,11 +16,10 @@
         "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
     },
     "dependencies": {
-        "botbuilder": "4.9.2",
-        "botbuilder-dialogs": "4.9.2",
-        "botbuilder-dialogs-adaptive": "4.9.2-preview",
-        "botbuilder-lg": "4.9.2",
-        "adaptive-expressions": "4.9.2",
+        "botbuilder": "4.11.0-rc1",
```
Comment (luhan2017) [inline experimental/adaptive-dialog/javascript_nodejs/01.multi-turn-prompt/package.json:19]: let's we wait for the final release and update the package version to that.

**Review by luhan2017 — APPROVED** [review]

File: experimental/adaptive-dialog/javascript_nodejs/01.multi-turn-prompt/package.json (hunk 1 above)
Comment (sgellock) [reply experimental/adaptive-dialog/javascript_nodejs/01.multi-turn-prompt/package.json:19]: agreed.  we don't update package.json's across the samples repo until after SDK's have been released.

**Comment by luhan2017** [comment]
--imports="../dialogs/imported" what does this mean?

**Review by stevengum — APPROVED** [review]
Thanks @chon219 for updating the samples and the SDK to make it simpler for customers to write code-first Adaptive Dialog bots!

**Comment by stevengum** [comment]
@chon219 could you answer @luhan2017's [question?](https://github.com/microsoft/BotBuilder-Samples/pull/2692#issuecomment-725916566)



> --imports="../dialogs/imported" what does this mean? I didn't see any folder generated for this.

**Comment by chon219** [comment]
> @chon219 could you answer @luhan2017's [question?](https://github.com/microsoft/BotBuilder-Samples/pull/2692#issuecomment-725916566)
> 
> 
> 
> > --imports="../dialogs/imported" what does this mean? I didn't see any folder generated for this.
> 
> 

This question is for Composer runtime and was pasted into this issue by mistake.

**Pull request closed without merging**